src/index.d.ts

```diff
@@ -0,0 +1,121 @@
+import * as React from 'react';
+
+export namespace ReactSpinners {
+	interface CommonProps {
+		color?: string;
+		loading?: boolean;
+	}
+
+	interface BarLoaderProps extends CommonProps {
+		height?: number | string;
```

Comment:
why is this `number` or `string`? The PropTypes for this loader asks for `number`.

Reply:
In case of only `number`, Typescript compiler will throw an error if you try to write something like `height="20"` in TSX markup. I thought it would be more convenient to allow this more HTML-like approach and not just write `height={ 20 }` all the time. But maybe you are right. I'll make necessary changes, so type definitions will strictly correspond to the docs.